style.css

```diff
@@ -0,0 +1,89 @@
+
+@media screen and (max-width: 768px) {
+  body {
+    background-color: #e5e5e5;
+    padding: 0;
+    margin: 0;
+  }
+
+  section {
+    background-image: url(./assets/images/header-shapes\ mobile.png);
+    background-size: cover;
+    background-repeat: no-repeat;
+    display: flex;
+    flex-direction: column;
+    object-fit: contain;
+    max-width: fit-content;
+    width: 100%;
+    height: 718px;
+  }
```

Comment:
- [ ] You've done a great job😍😍 so far! 
- [ ] However, using a fixed height seems to be causing some issues with your design's responsiveness. 
- [ ] To ensure your design follows a responsive strategy, I suggest using percentages instead. This will allow your design to adapt to different screen sizes effectively. 

- [ ] Furthermore, I noticed that the website you made does not match the given Figma design. 🤔🎉
- [ ] It's important to adhere to the Figma design and project requirements to ensure that your design looks visually appealing and catches the attention of potential recruiters in the future.🥱🥱

- [ ] your design
![screencapture-raw-githack-kakalygaday-Portfolio-Project-Project-index-html-2023-07-16-01_51_08](https://github.com/kakalygaday/Portfolio-Project/assets/118761303/539f1f47-3f0e-43d6-9192-38da41c4bb87)